coderadar-vcs/src/main/java/io/reflectoring/coderadar/vcs/adapter/ExtractProjectCommitsAdapter.java

```diff
@@ -81,6 +81,7 @@ private Commit mapRevCommitToCommit(RevCommit rc) {
     Commit commit = new Commit();
     commit.setName(rc.getName());
     commit.setAuthor(rc.getAuthorIdent().getName());
+    commit.setAuthorEmail(rc.getAuthorIdent().getEmailAddress());
```

Comment:
The Jgit javadoc states the following:

```
	 * This method parses and returns the content of the author line, after
	 * taking the commit's character set into account and decoding the author
	 * name and email address. This method is fairly expensive and produces a
	 * new PersonIdent instance on each invocation. Callers should invoke this
	 * method only if they are certain they will be outputting the result, and
	 * should cache the return value for as long as necessary to use all
	 * information from it.
```

You can save the AuthorIdent to a variable and the use that to get the name and email (and timestamp as well)

Reply:
Done.